src/components/Section/History/Employment/EmploymentItem.jsx

```diff
@@ -453,6 +453,13 @@ export default class EmploymentItem extends ValidationElement {
             required={this.props.required}
             scrollIntoView={this.props.scrollIntoView}
           />
+          {this.props.render({
+            address: this.props.SupervisorAlternateAddress,
+            allowForeignMilitary: false,
```

Comment:
I guess I didn't look at this carefully in the previous PRs, but I'm not following the workflow characteristic that this controls. Flipping it back and forth from false/true didn't seem to have an effect on the supervisor primary/secondary address UI.

Reply:
the `allowForeignMilitary` switch should give us finer-grained control over whether or not we show the branch option in this component. So, it would be a bug if we actually wanted to conditionally show the branch!

It sounds like we can just remove it based on the comment above?

Reply:
@el-mapache Yeah, I wasn't saying it being false was a bug – I was just saying that I played with the UI with it set to false, and then set it to true, re-built the app, and didn't notice any behavior change. i.e., the behavior I was expecting by a quick read of the code wasn't appearing to happen in the UI.

If you've removed it at this point, it is moot though :)